TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/Drivetrain.java

```diff
@@ -1,26 +1,32 @@
 package org.firstinspires.ftc.teamcode.subsystems;
 
 import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.IMU;
 
 import org.firstinspires.ftc.teamcode.hardware.RobotHardware;
 
 public class Drivetrain{
-    private final DcMotor motorFrontRight;
-    private final DcMotor motorFrontLeft;
-    private final DcMotor motorBackRight;
-    private final DcMotor motorBackLeft;
-
+    private final DcMotor motorRight;
+    private final DcMotor motorLeft;
+    private final IMU imu;
     public Drivetrain(){
-        motorFrontRight = RobotHardware.motorFrontRight;
-        motorFrontLeft= RobotHardware.motorfrontLeft;
-        motorBackLeft= RobotHardware.motorbackLeft;
-        motorBackRight = RobotHardware.motorbackRight;
+        imu = RobotHardware.DTimu;
```

Comment:
suggestion:
considering this a static value, use as a static import isntead of adding to a variable